.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
+  # Pre-reqs for building omsagent
+  - sudo apt-get install -qq git pkg-config make g++ rpm librpm-dev libpam0g-dev libssl-dev libkrb5-dev curl libcurl4-gnutls-dev  
+
+# We are in /home/travis/build/Microsoft/Powershell-DSC-for-Linux/
+
+
+install: 
+  # Clone OMI
+  - cd ..
+  - git clone https://github.com/Microsoft/omi.git
+  - cd -
+  - ln -s ../omi/Unix ./omi-1.0.8
+
+before_script:
+  # PR validation during check-in will be setup in github in the settings section of our repo
+  - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then git fetch origin refs/pull/$TRAVIS_PULL_REQUEST/head ; git checkout FETCH_HEAD; fi 
```

Comment:
I'm not sure why this is there. You shouldn't need it if you're using the actual repo that was given to you, as I do when I deal with a superproject in SCXcore.